webpack.config.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
   context: path.resolve(__dirname, './src'),
   target: 'web',
   entry: [
+    'babel-polyfill',
```

Comment:
Got the error described here: http://mysticcoders.com/blog/fixing-regenerator-runtime-error-integrating-redux-saga
Now importing it where it should be. Thanks!